docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture-use-cases.md

```diff
@@ -0,0 +1,163 @@
+- **Message Center**: One single [Message Center](https://help.vtex.com/en/tutorial/conhecendo-o-message-center--tutorials_84) for all stores.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/multi-currency-language-single-account.png)
+
+#### Why choose this architecture?
+
+- Supports [multi-currency](https://developers.vtex.com/docs/guides/vtex-io-documentation-internationalizing-product-prices) natively.
+- Supports multi-language with [Messages](https://developers.vtex.com/docs/guides/storefront-content-internationalization) and [Catalog Translation App](https://developers.vtex.com/docs/guides/catalog-internationalization).
+
+>⚠ Product names are not translated on the [Checkout](https://help.vtex.com/en/tutorial/checkout-vtex-overview--7wcprkM7yZUflOqbzAN5SI) and [My Account pages](https://help.vtex.com/en/tutorial/how-my-account-works--2BQ3GiqhqGJTXsWVuio3Xh?&utm_source=autocomplete), as these modules call directly the catalog API to get product data, so the information does not pass through the Messages API.
+
+This model is easier to set up, integrate, and maintain compared to the [multi-account](#multi-account) approach, but it may not be efficient for large and complex operations. This architecture is recommended for operations present in more than one country, requiring the configuration of different languages and currencies, but without the need for data segregation, as all core services are shared. Therefore, it may work well for operations managed by a single, centralized team.
+
+### Multi-account
+
+In this architecture, there is a main VTEX account that will act as a [seller](https://help.vtex.com/en/tutorial/o-que-e-um-seller--5FkLvhZ3Few4CWWIuYOK2w) in all secondary VTEX accounts, acting as [marketplaces](https://help.vtex.com/en/tutorial/configurando-marketplace-entre-lojas-vtex). The main account - the seller - will be the source of truth for the [VTEX Core services](https://developers.vtex.com/docs/guides/getting-started#vtex-core-services), such as catalog, pricing, logistics, etc.
+
+The reference architecture below exemplifies its main characteristics:
+
+- **Separated VTEX account**: Each store has its own VTEX account.
+- **Separated website**: Each store has its own website and may customize its frontend with a different look and feel per store.
+- **Customer Data Storage**: Segregated by marketplaces' account. There is no customer information within the main account’s [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction).
+- **Catalog**: Separated catalog management, providing a unique assortment per store, defined based on the main account’s catalog.
+- **Logistics and Order Management System (OMS)**: Independent Logistics and OMS per store.
+- **Fulfillment**: Supports single or multiple fulfillment.
```

Comment:
What is single or multiple fulfillment?

Reply:
In this multi-account scenario, a single order can consist of items from one or multiple accounts. I will make this clearer in the document.